angular-workspace/components/expandable/expandable.decorator.ts

```diff
@@ -11,7 +11,10 @@ export const decorator: Addon_DecoratorFunction<StoryFnAngularReturnType> = (sto
       ${template}
 
       <style>
-        dso-expandable {
+        dso-expandable[open] {
+          border: 1px solid black;
+        }
+        dso-expandable:not(.dso-hide) {
           border: 1px solid black;
```

Comment:
- deze kunnen zo te zien gecombineerd worden.
- we gebruiken geen color keywords; in dit geval dus #000